releasenotes/notes/switch-syntax-refactor-dd5253d79bc695aa.yaml

```diff
@@ -0,0 +1,5 @@
+---
+features:
+  - |
+    Added support for switch statements, allowing for compact description of multi-way
+    branches that depend on a common control expression.
```

Comment:
This file should probably have a trailing new line.

Reply:
Fixed in beda3de